UBI7-init/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi7-init
+MAINTAINER https://github.com/JacobCallahan
```

Comment:
This could be you as well :)

Reply:
Meh, not interested unless this repository is transferred to the SatelliteQE GitHub organization. :smile: But thank you.